tests/integration-tests/configs/develop.yaml

```diff
@@ -5,9 +5,9 @@ test-suites:
 {% include 'common/common.yaml' %}
 {% endfilter %}
   runtime_bake:
-    test_runtime_bake.py::test_runtime_bake:  # These are currently skipped
+    test_runtime_bake.py::test_runtime_bake: # arm instance tests are temporarily skipped
       dimensions:
-        {%- for os, scheduler in [("alinux2", "slurm"), ("ubuntu2004", "torque"), ("centos7", "sge")] %}
+        {%- for os, scheduler in [("alinux2", "torque"), ("centos7", "sge"), ("centos8", "slurm"), ("ubuntu2004", "torque")] %}
```

Comment:
was there a reason for changing this so that AL2 runs with torque instead of slurm?

Reply:
No need to change it. I was just wondering.